addons/l10n_hu_edi/models/account_move.py

```diff
@@ -327,7 +327,7 @@ def _l10n_hu_edi_check_invoices(self):
             'company_vat_invalid': {
                 'records': self.company_id.filtered(
                     lambda c: (
-                        (c.vat and not hu_vat_regex.fullmatch(c.vat))
+                        (c.account_fiscal_country_id.code == 'HU' and c.vat and not hu_vat_regex.fullmatch(c.vat))
```

Comment:
since the `not hu_vat_regex.fullmatch(...)` check is getting more common, how about we move it to another function with better name? like `_l10n_hu_edi_vat_valid(...)`